### PR TITLE
Add version consistency test

### PR DESCRIPTION
**Pull request by aliok**
Fixes https://github.com/knative/eventing-operator/issues/23

### CODE REVIEW

File: version/version_test.go [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	resources := m.Resources
+
+	for _, resource := range resources {
+		v, found := resource.GetLabels()["eventing.knative.dev/release"]
+		if !found {
+			// label is missing for this resource. we cannot do the check.
+			continue
+		}
+		if v != expectedLabelValue {
+			t.Errorf("Version info in manifest and operator don't match. got: %v, want: %v. Resource GVK: %v, Resource name: %v", v, expectedLabelValue,
```
Comment (aliok) [inline version/version_test.go]: If version info doesn't match, it shows this message multiple times.
Alternatively, we can switch to failing eagerly with `Fatalf`

**Comment by aliok** [comment]
cc @houshengbo @matzew

File: version/version.go [hunk 2]
```diff
@@ -2,5 +2,5 @@ package version
 
 var (
 	// Version is the version that will be applied to the KnativeServing resource.
-	Version = "0.10.0"
+	Version = "0.10.1"
```
Comment (matzew) [inline version/version.go]: nice catch ;-)

**Comment by aliok** [comment]
@matzew need a new /lgtm . Fixed the test config error.

**Pull request merged**